force-app/main/default/aura/WorkItemBoard/WorkItemBoardController.js

```diff
@@ -0,0 +1,22 @@
+({
+    doInit : function(component, event, helper) {
+        var action = component.get("c.retrieveWIData"); //component.get("c.loadWIStatusColumns");
+        
+        action.setCallback(this, function(response) {
+            var state = response.getState();
+            
+            if(state === "SUCCESS") {
+                var result = response.getReturnValue();
+                console.log(result.wItems);
```

Comment:
remember to remove unnecessary logs from the code to be uploaded into the repository.